apps/pom.xml

```diff
@@ -573,6 +574,20 @@
                 <version>${commons-codec.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
do we really need POI at the root level?

Reply:
This is in the dependencyManagement section so it doesn't actually add the dependency globally.